src/components/NewMessage/NewMessage.vue

```diff
@@ -527,8 +527,8 @@ export default {
 
 			if (this.upload) {
 				// Clear input content from store
-				this.$store.dispatch('setCurrentMessageInput', { token: this.token, text: '' })
-				this.$store.dispatch('removeMessageToBeReplied', this.token)
+				this.chatExtrasStore.setChatInput({ token: this.token, text: '' })
+				this.chatExtrasStore.removeParentIdToReply(this.token)
```

Comment:
Conflicting diff with main branch, line is there from https://github.com/nextcloud/spreed/commit/3f443b3e0b4fa6e550f8c8d21bdc0eea420e1565 backport